README.md

```diff
@@ -48,6 +49,15 @@ filesystem with some added configurations and packages. It is ~50mb squashed
 and occupies ~120mb of space when mounted. The base image[s] are used as a shared
 image that different overlays can be built and mounted with, and take 3-5 minutes to build.
 
+**base image large**
```

Comment:
maybe it's not so clear with 'base image large'?  maybe it could be 'base image' here, and above 'base image' change to 'minbase image', just a thought that it could align with debootstrap 's 'minbase' arguments, because it's the only difference. or another better semantic is also fine.